docs/source/tuning.rst

```diff
@@ -125,6 +125,60 @@ Experimenting with this value yields a better result at ``2e2``.
 Here, the scan matcher used rotation to still slightly mess up the result though.
 Setting the ``rotation_weight`` to ``4e2`` leaves us with a reasonable result.
 
+Special Cases
+-------------
+
+The default configuration and the above tuning steps are focused on quality.
+Only after we have achieved good quality, we can further consider low latency tasks and other special cases.
+
+Low Latency
+^^^^^^^^^^^
+
+Low latency is required for online algorithms, such as robot localization.
+Local SLAM, which operates in the foreground, directly affects latency.
+Global SLAM builds up a queue of background tasks. When global SLAM cannot keep up the the queue,
+drift can accumulate indefinitely, so global SLAM should be tuned to work in real time.
+
+To tune the different components for speed, there are many options, and we list them ordered from 
+the recommended, effective ones to the those that are more intrusive.
+It is recommended to only explore one option at a time, starting with the first.
+
+To tune global SLAM for lower latency, we can
```

Comment:
Add a sentence here that explains that there is a certain threshold, that is not easy to determine: you can crank everything up as high as possible, which means you are slow but you have the best quality. The goal now is to go as low as possible without loosing quality, but gaining performance. You never want to do more work than this.

And if needed you can cross over and trade quality for performance.

Reply:
Done